### PR TITLE
ENG-1136 BoolForm

### PR DESCRIPTION
**Pull request by aletex1994**
- Added BoolForm > boolsForm.tsx 
- Added BoolForm > converters.ts - function to check if the passed string is a valid bool value, if it is, it return a CandyBool type 
- Added  BOOLEAN to the Array constants.ts > VALIDATION_ERRORS 
- Added bool to constants.ts > SELECT_OPTIONS 
- Updated the formtypes > index.tsx with the new BoolsForm layout
- Added unit test for BoolForm > converters.ts

### CODE REVIEW

File: packages/candy_editor/src/constants.ts [hunk 1]
```diff
@@ -26,4 +27,5 @@ export const VALIDATION_ERRORS: { [key: string]: string } = {
   INT16: 'The value must be a Int number between -32768 and 32767',
   INT32: 'The value must be a Int number between -2147483648 and 2147483647',
   INT64: 'The value must be a Int number between -9223372036854775808 and 9223372036854775807',
+  BOOLEAN: 'The value must be a Boolean (true or false - case insensitive)',
```
Comment (Motokoder) [inline packages/candy_editor/src/constants.ts:30]: These should not be uppercase. Only the object name VALIDATION_ERRORS needs to represent that the property values are constants.

File: packages/candy_editor/src/assets/formTypes/index.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { CandyClassEditor } from '../../types';
 import { TextForm } from './TextForm/textForm';
 import { NaturalsForm } from './NatForms/naturalsForm';
 import { IntegersForm } from './IntForms/integersForm';
+import { BoolsForm } from './BoolForm/boolsForm';
```
Comment (Motokoder) [inline packages/candy_editor/src/assets/formTypes/index.tsx:6]: Please change to BoolForm.

File: packages/candy_editor/src/assets/formTypes/BoolForm/converters.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { CandyBool } from '../../../types';
+
+export function convertToBool(selectedValue: string): CandyBool | undefined {
```
Comment (Motokoder) [inline packages/candy_editor/src/assets/formTypes/BoolForm/converters.ts:3]: Please change to convertToCandyBool and update all other converter functions to include Candy.

**Review by Motokoder — CHANGES_REQUESTED** [review]

File: packages/candy_editor/src/constants.ts (hunk 1 above)
Comment (aletex1994) [reply packages/candy_editor/src/constants.ts]: Done, removed the period in Nat as well

**Review by Motokoder — APPROVED** [review]

**Pull request merged**